src/client/sandbox/style.js

```diff
@@ -225,6 +225,11 @@ export default class StyleSandbox extends SandboxBase {
             }
         });
 
+        window.CSSStyleSheet.prototype.insertRule = function (value, index) {
+            value = styleProcessor.process(value, getProxyUrl);
+            nativeMethods.styleSheetsInsertRule.call(this, value, index);
```

Comment:
Please add the blank line before the native method call and return the result of the native method execution.
Also, the returned value should be tested :)
https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet/insertRule#Return_value